src/asmjs/asm_v_wasm.cpp

```diff
@@ -99,14 +99,13 @@ FunctionType* ensureFunctionType(std::string sig, Module* wasm) {
     return wasm->getFunctionType(name);
   }
   // add new type
-  auto type = new FunctionType;
+  std::unique_ptr<FunctionType> type{new FunctionType};
```

Comment:
Thanks for the PR!

Overall I agree it makes sense to use unique_ptrs for stuff like this. However, we use `auto* thing = make_unique<Type>()` in other places, so we should be consistent one way or the other with the rest of the code. I don't have a strong opinion either way myself.

Reply:
I've seen this, but `make_unique` is not included in every file I modified. I'm happy to change this accordingly.

Reply:
Might need to add an include for it. I think it's in `src/support/utilities.h`.

Reply:
This is fixed already.